lib/spotify.js

```diff
@@ -1,21 +1,37 @@
-const scopes = [
-  "user-read-email",
-  "user-read-private",
-  "streaming",
-  "user-read-playback-state",
-  "user-modify-playback-state",
-  "user-read-currently-playing",
-  "user-read-recently-played",
-  "user-library-read",
-  // "user-library-modify",
-].join(",");
+import { Buffer } from "buffer";
+import axios from "axios";
 
-const params = {
-  scope: scopes,
-};
+// https://www.reddit.com/r/nextjs/comments/10o6aup/next_auth_spotify_reauthentication_access_token/
 
-const queryParamString = new URLSearchParams(params);
+const SPOTIFY_REFRESH_TOKEN_URL = "https://accounts.spotify.com/api/token";
 
-const LOGIN_URL = `https://accounts.spotify.com/authorize${queryParamString.toString()}`;
-
-export { LOGIN_URL };
+export async function refreshAccessToken(token) {
+  try {
+    const basicAuth = Buffer.from(
+      `${process.env.SPOTIFY_CLIENT_ID}:${process.env.SPOTIFY_CLIENT_SECRET}`
+    ).toString("base64");
```

Comment:
If you only need a base64 string, you could use the [`btoa()` function](https://developer.mozilla.org/en-US/docs/Glossary/Base64) instead of using the buffer as well.

Reply:
alright, I will look into it

Reply:
When I try to write it with `btoa()` my linter says it is deprecated and I should use the options above:

![Bildschirm­foto 2023-02-14 um 13 05 40](https://user-images.githubusercontent.com/117563487/218733461-19ac860e-bcfa-4021-b874-2ca749be9123.png)